data/ecosystem/vendors.yaml

```diff
@@ -306,6 +306,12 @@
   contact:
   oss: false
   commercial: true
+- name: Parseable
+  nativeOTLP: true
+  url: https://www.parseable.com/docs/server/opentelemetry
+  contact:
```

Comment:
@Debanitrkl is there a contact field we can list here?

Reply:
Yeah sure will add a contact field